main.tf

```diff
@@ -86,9 +78,7 @@ resource "aws_default_security_group" "this" {
   }
 
   tags = merge(
-    {
-      "Name" = format("%s", var.default_security_group_name)
-    },
+    { "Name" = coalesce(var.default_security_group_name, "${var.name}-default") },
```

Comment:
but this is one of the reasons for the PR - by default the default security group name is "" which leads to unnecessary changes https://github.com/terraform-aws-modules/terraform-aws-vpc/issues/684

I would think a tag name change from `""` -> `"<name>-default"` would be acceptable and not considered breaking, no?

Reply:
said differently - when Hashicorp adds tags to resources, thats not a breaking change just like here, if a custom name is not provided we are setting a default one to avoid unnecessary updates in plans (and if users are concerned with the name, they have set a value already via the variable and so its a no-op for them)

Reply:
fixed in https://github.com/terraform-aws-modules/terraform-aws-vpc/pull/730/commits/7de6785d09de2a6f55f8c6a214909977d5d3f00a

Reply:
Great! 👍